pages/babylonView.js

```diff
@@ -50,6 +64,10 @@ window.BabylonView = function() {
         scene = new BABYLON.Scene(engine);
         scene.useRightHandedSystem = true; // This is needed for correct glTF normal maps.
 
+        BABYLON.SceneLoader.OnPluginActivatedObservable.add(function (plugin) {
+            plugin.animationStartMode = BABYLON.GLTFLoaderAnimationStartMode.NONE;
+        });
```

Comment:
Might consider doing this also:

```javscript
    BABYLON.SceneLoader.OnPluginActivatedObservable.add(function (plugin) {
        plugin.animationStartMode = BABYLON.GLTFLoaderAnimationStartMode.NONE;
    }, undefined, undefined, undefined, true);
```

This will make the observable unregister itself after it fires.